plugins/modules/zos_fetch.py

```diff
@@ -20,21 +20,18 @@
 version_added: "2.9"
 short_description: Fetch data from z/OS
 description:
-  - This module copies a file or data set from a remote z/OS system to the
-    local machine. Use the
-    L(fetch,https://docs.ansible.com/ansible/latest/modules/fetch_module.html)
-    module to copy files from local machine to the remote z/OS system.
+  - This module fetches a file or data set from a remote z/OS system. 
```

Comment:
I think we should be a bit more verbose and give credit to the details:

This module fetches a Unix System Services (USS) file,
PS(sequential data set), PDS, PDSE, member of a PDS/PDSE or
KSDS(VSAM data set).